.github/workflows/downstream_protobuf_compatibility_check_nightly.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
           - java-storage-nio
         # Default Protobuf-Java versions to use are specified here. Without this, the nightly workflow won't know
         # which values to use and would resolve to ''.
-        protobuf-version: ${{ fromJSON(format('[{0}]', inputs.protobuf_runtime_versions || '"3.25.5","4.29.0"')) }}
+        protobuf-version: ${{ fromJSON(format('[{0}]', inputs.protobuf_runtime_versions || '"3.25.5","4.32.0"')) }}
```

Comment:
Can we also update the 3.x default to `3.25.8`?

Reply:
Done. Thanks!